src/playbacks/no_op/no_op.js

```diff
@@ -25,13 +25,16 @@ export default class NoOp extends Playback {
     this.$el.html(this.template({message: playbackNotSupported}))
     this.$el.append(style)
     this.trigger(Events.PLAYBACK_READY, this.name)
-    this._options.autoPlay && this.play()
+    const showForNoOp = !!(this.options.poster && this.options.poster.showForNoOp);
+    if (this.options.autoPlay || !showForNoOp) {
+      this.play()
+    }
```

Comment:
I think it would be better to have the animation always running than have the playback have to look at poster options as the playback and the poster plugin should be completely independent. Also someone might not be using the poster plugin in which case they would also expect it to be animating.

I.e call `this._animate()` in the `render()` method. (I think the `render()` method in a playback is only called once)